src/screens/Passcode/EnterPasscodeContainer.js

```diff
@@ -96,10 +99,15 @@ class EnterPasscodeContainer extends NavigatorComponent<Props & Actions & Settin
     }
   };
 
+  onForgetPasscode = () => {
+    this.props.onForget();
+  }
+
   renderPasscodeScreen() {
     const {
-      navigator, passcodeType, title, onCancel,
+      navigator, passcodeType, title, onCancel, isLoggedIn,
     } = this.props;
+    const shouldShowForget = !isLoggedIn && this.props.accountId != null;
```

Comment:
Please use `isLoggedIn === false` instead of `!isLoggedIn` to prevent coercion.

Reply:
done